src/main/java/com/google/devtools/build/lib/bazel/rules/android/android_ndk_misc_libraries_template.txt

```diff
@@ -1,6 +1,6 @@
 cc_library(
     name = "cpufeatures",
-    srcs = glob(["ndk/sources/android/cpufeatures/*.c"]),
-    hdrs = glob(["ndk/sources/android/cpufeatures/*.h"]),
+    srcs = glob(["ndk/sources/android/cpufeatures/*.c"], allow_empty = True),
```

Comment:
It doesn't make sense to me that these could ever result in failed globs in real life, but maybe they do in our tests.
OTOH, I don't think it is a problem that we hide the glob failure. We will simply push the error from glob time until link time, but the build will still fail, so all is good.

The same question applies everywhere.